.gitlab-ci.yaml

```diff
@@ -0,0 +1,12 @@
+stages:
+  - integration-test
+
+polkadot-integration-test:
+  stage: integration-test
+  image: paritytech/ci-unified:latest
+  tags:
+    - zombienet-polkadot-integration-test
+  script:
+    - kubectl get ns
```

Comment:
I don't think `ci-unified` have `kubectl` installed, we also don't need it. Maybe I can write an small script to test the access to the cluster easily.
Thx!